R/cf.R

```diff
@@ -400,10 +405,19 @@ addConfIndex2cf <- function(cf, conf.index) {
 }
 
 addStat.cf <- function(cf1, cf2) {
+  stopifnot(inherits(cf1, 'cf'))
+  stopifnot(inherits(cf2, 'cf'))
+
+  if (is_empty.cf(cf1)) {
+    return (invisible(cf2))
+  }
+  if (is_empty.cf(cf2, 'cf_orig')) {
```

Comment:
why two arguments?

Reply:
That was just negligence, it is fixed now.